mithril-test-lab/mithril-end-to-end/README.md

```diff
@@ -34,6 +34,28 @@ make build
 ./mithril-end-to-end --db-directory db/ --bin-directory ../../target/release
 ```
 
+> [!WARNING]
```

Comment:
Maybe just a **### Note for MacOS users** instead of a **[!WARNING]**.

Can you can also remove all the `>`.